supply-chain/config.toml

```diff
@@ -641,6 +637,10 @@ criteria = "safe-to-deploy"
 version = "0.2.13"
 criteria = "safe-to-deploy"
 
+[[exemptions.redox_syscall]]
+version = "0.3.5"
```

Comment:
Could you check to see if 0.2.13, which is exempted, is a small or big diff from 0.3.5? If small mind adding an audit for that? If big though it's a bit of a bummer but we can probably live with an exemption.

Reply:
I didn't want to do any sort of audit on this because it is the redox syscall layer, an OS I don't know anything about and don't think we care about in the context of this project. It just happens to be a transitive dep in some crates.

Reply:
Ok sounds reasonable 👍